utils/package.json

```diff
@@ -1,13 +1,15 @@
 {
-    "name": "utilities-mock-data-loader",
+    "name": "dtfs-utilities",
     "version": "1.0.0",
     "description": "DTFS Utilities",
     "license": "MIT",
     "author": "UK Export Finance",
     "scripts": {
+        "predb:seed": "rm -rf sql-db-seeder/node_modules && (cd sql-db-seeder && mkdir node_modules && cp -r ../../node_modules/typeorm-extension node_modules)",
+        "db:seed": "npx ts-node sql-db-seeder/node_modules/typeorm-extension/bin/cli.cjs seed:run -r sql-db-seeder -d src/seeding-data-source.ts",
```

Comment:
Please can we update the docs to mention you need to run `predb:seed` and maybe also a short explanation of this weirdness?